src/ui/src/components/chart/FunnelChart.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {
+  JinjatECharts,
+  JinjatEChartsProps,
+} from "@components/chart/JinjatECharts";
+
+interface JinjatFunnelChartsProps extends JinjatEChartsProps {
+  data: undefined;
+  nameCol: undefined;
+  valueCol: undefined;
+}
+
+export const JinjatFunnelChart: React.FC<JinjatFunnelChartsProps> = (props) => {
+  const options = props.options || {};
+  if (options.series != null) {
+    return (
+      <div>
+        `options.series` is not null, please use `ECharts` component instead.
+      </div>
+    );
+  }
+  options.series = [
+    {
+      type: "funnel",
+      encode: {
+        data: props.data,
+        nameCol: props.nameCol,
```

Comment:
are you sure that this is valid? I don't see the `nameCol` in Echarts config: https://echarts.apache.org/en/option.html#series-funnel.encode

Reply:
I need clarification on this. I see these properties in the Evidence example, but as you said, not in the Echarts config. I am unsure about how can I find the actual property from Echarts API docs that match with the Evidence example.